hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -432,45 +433,55 @@
     {
       name: chip_rv_dm_cpu_debug_mem
       desc: '''Verify access to the debug mem from the CPU.
+
+            - CPU programs RW type CSRs (such as progbuf0 and progbuf1) with random value.
+            - Read back to check their values for correctness.
             '''
       milestone: V2
       tests: []
     }
     {
       name: chip_rv_dm_jtag_debug_mem
       desc: '''Verify access to the debug mem from the external JTAG interface.
+
+            - Load debug ROM and read back through jtag to check the values for correctness.
             '''
       milestone: V2
       tests: []
     }
     {
       name: chip_rv_dm_cpu_debug_req
```

Comment:
Due to observability constraints, this test is now expanded to test a full debug cycle. So let's name it accordingly, e.g. "chip_rv_dm_perform_debug" or so.

Reply:
sg, renamed it.